blib2to3/Grammar.txt

```diff
@@ -138,8 +138,8 @@ arglist: argument (',' argument)* [',']
 # that precede iterable unpackings are blocked; etc.
 argument: ( test [comp_for] |
             test '=' test |
-	    '**' expr |
-	    star_expr )
+	    '**' test |
+            '*' test )
```

Comment:
Yup, that's what CPython does.

Reply:
where does this happen in CPython?

Reply:
https://github.com/python/cpython/blob/master/Grammar/Grammar#L136-L139